### DISCUSSION TITLE
Why some props is red in template?

### DISCUSSION THREAD

**Discussion by kingyue737** [vuejs/language-tools#1363] [Q&A] [answered]
I'm using vuetify2 with volar. The `align` prop of `v-row` is red.

![image](https://user-images.githubusercontent.com/40021217/170861440-8097e905-8210-4b0d-862e-c7ff03d4cd74.png)

Besides, the slash below is also red.

![image](https://user-images.githubusercontent.com/40021217/170861459-33bb65c4-731a-4bfa-aa7b-04121a22c862.png)

**Comment by johnsoncodehk** [comment, marked as answer]
See https://github.com/johnsoncodehk/volar/issues/1317#issuecomment-1140383682.

(`align` token is `invalid.deprecated.entity.other.attribute-name.html`)
